### ISSUE TITLE
polyhedron draw limited when the polyhedron is planar

### ISSUE THREAD

**Issue by boutil** [CGAL/cgal#4966]
_Please use the following template to help us solving your issue._

## Issue Details

I am using cgal 5.0.3 in Debian and want to draw a polyhedron made of glued rhombi which turns out to be planar.
Whereas it was rendered correctly with the version 4.x, it becomes suboptimal with this version:
- the only available movement is a rotation in the plane containing the polyhedron (which is thus always the one of the screen)
- internal edges are not rendered, and one can see only the boundary.
As soon as I add an extra triangle outside of that plane, all the 3d movements are available and the edges are rendered correctly. 

## Source Code

Here is a minimal example where the problem is present: two coplanar triangles are shown as a single blue square.

```c++
#include <CGAL/Simple_cartesian.h>
#include <CGAL/Polyhedron_3.h>
#include <CGAL/draw_polyhedron.h>

#include <iostream>

typedef CGAL::Simple_cartesian<double>               Kernel;
typedef Kernel::Point_3                              Point_3;
typedef CGAL::Polyhedron_3<Kernel>                   Polyhedron;
typedef Polyhedron::Facet_iterator                   Facet_iterator;
typedef Polyhedron::Halfedge_around_facet_circulator Halfedge_facet_circulator;

int main() {
    Point_3 p( 0.0, 0.0, 0.0);
    Point_3 q( 1.0, 0.0, 0.0);
    Point_3 r( 0.0, 1.0, 0.0);
    Point_3 s( 1.0, 1.0, 0.0);

    Polyhedron P;

    P.make_triangle(p,q,r);
    P.make_triangle(q,s,r);
    P.make_triangle(p,q,t);

    CGAL::draw(P);

    return EXIT_SUCCESS;
}
```
adding a point `Point_3 t( 0.0, 0.0, 0.1)` and a triangle (p,q,t) gives an example where 3D rotation works and the edge between the two coplanar triangles is rendered correctly (after a slight move).

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): GNU/Linux Debian unstable
* Compiler: g++ 10.2.0
* Release or debug mode:
* Specific flags used (if any):
* CGAL version: 5.0.3
* Boost version: 1.71.0

**Comment by maxGimeno** [comment]
When displaying a pure 2D data, the 3D movements are disabled to only keep useful manipulation. 
For the edges problem, it seems it has been fixed in 5.1, soom to be released. @gdamiand, do you confirm ?

**Comment by gdamiand** [comment]
> When displaying a pure 2D data, the 3D movements are disabled to only keep useful manipulation.
> For the edges problem, it seems it has been fixed in 5.1, soom to be released. @gdamiand, do you confirm ?

Exactly @maxGimeno !

**Comment by boutil** [comment]
Thank you so much @maxGimeno and @gdamiand .
I am looking forward for 5.1 then!

As for the 2D/3D movements, it indeed makes sense to lock the supposedly unneeded movements when showing pure 2D data and it is certainly the most useful choice. Being able to unlock them would be nice in order to enjoy the nice shading effects, but that is purely cosmetic.

**Comment by maxGimeno** [comment]
@gdamiand would it be complicated to add a key binding to disable the lock ?

**Comment by gdamiand** [comment]
> @gdamiand would it be complicated to add a key binding to disable the lock ?

Very easy; I will do asap.

**Comment by maxGimeno** [comment]
@gdamiand thank you !

**Comment by boutil** [comment]
you're awesome!

**Comment by gdamiand** [comment]
Done in #4968.

**Comment by boutil** [comment]
I confirm that the edge drawing issue for planar polyhedra is fixed with 5.1.
